packager/hls/base/media_playlist.cc

```diff
@@ -15,12 +16,17 @@
 #include "packager/base/logging.h"
 #include "packager/base/strings/string_number_conversions.h"
 #include "packager/base/strings/stringprintf.h"
+#include "packager/base/time/time.h"
 #include "packager/file/file.h"
 #include "packager/hls/base/tag.h"
 #include "packager/media/base/language_utils.h"
 #include "packager/media/base/muxer_util.h"
 #include "packager/version/version.h"
 
+DEFINE_bool(hls_ext_x_program_date_time,
+            false,
+            "hls print EXT-X-PROGRAM-DATE-TIME tag");
```

Comment:
Can you define it in https://github.com/google/shaka-packager/blob/master/packager/app/hls_flags.cc instead?

And then add a bool in HlsParams?

https://github.com/google/shaka-packager/blob/master/packager/hls/public/hls_params.h

Reply:
Ping

Reply:
Done.